src/modules/ekf2/EKF2Selector.hpp

```diff
@@ -180,6 +182,11 @@ class EKF2Selector : public ModuleParams, public px4::ScheduledWorkItem
 	bool _gyro_fault_detected{false};
 	bool _accel_fault_detected{false};
 
+	static constexpr int8_t VIS_MODE_INVALID{-1};
+	static constexpr int8_t VIS_MODE_NORMAL{0};
```

Comment:
3rd location of same definitions. Possible to reduce?

Reply:
Removed from here now, defined in estimator_status msg